pytest-extension/README.md

```diff
@@ -0,0 +1,38 @@
+# pytest-exasol-extension Plugin
+
+The `pytest-exasol-extension` plugin provides a pytest fixtures for preparing a database for the extension tests.
+The fixtures are backend agnostic. They run for the selected backends
+(see the documentation for the `pytest-exasol-backend` plugin).
+
+## Installation
+
+The pytest-exasol-extension plugin can be installed using pip:
+
+```shell
+pip install pytest-exasol-extension
+```
+
+## Usage in Tests
+
+Below is an example of a test that requires a database connection with an open test schema.
+
+```python
+import pytest
+
+@pytest.fixture(scope="session")
+def db_schema_name() -> str:
```

Comment:
This fixture is not used directly by the tests below. 
Is the fixture required by pyexasol_connection or by the backend plugin? 
If so, then please add a comment, e.g. "This fixture is required by ...".